README.md

```diff
@@ -1,2 +1,27 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+likeliness = 0
+count = 0
+
+print 'please enter your word'
+read word
+character = char{n}
+compare char {0} # compares the first character of the input with the solution
+
+if char{0} = true then :
```

Comment:
Seems strange to compare a character to a boolean `true` -- would be better to have seomthing like `if guess[0] == secret[0]`